lib/common/formats.js

```diff
@@ -34,7 +40,7 @@ function variablesWithPrefix(prefix, properties) {
 }
 
 function iconsWithPrefix(prefix, properties, config) {
-  return fileHeader() + _.chain(properties)
+  return fileHeader(config.options) + _.chain(properties)
```

Comment:
`config` here is referencing the platform object and not the file object, like `this` refers to inside the formatter function (like on line 74 below). Should we change the 3rd argument to just be 'options' and then the formatter passes only the options object? I'm assuming we are only going to do file level options for this PR and do the cascading options in a later one.

Reply:
Now that I look at it, our iconsWithPrefix function is a bit wonky and confusing. Maybe for now, what if we removed the `fileHeader()` in iconsWithPrefix and moved it to the formatter functions themselves so it matches variablesWithPrefix ?

Reply:
yes, great idea!